.github/workflows/workflow.yml

```
@@ -84,3 +87,26 @@ jobs:
                          --git.pushArgs=--follow-tags
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      - run: echo "Released ${{ steps.version.latest }} :rocket:" >> $GITHUB_STEP_SUMMARY
+
+  publish:
+    name: Publish
+    needs: release
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: latest
+          path: dist
+      - name: Build images
+        run: docker compose build"
```

I think the " isn't needed?

```suggestion
        run: docker compose build
```